lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1595,9 +1600,10 @@ def _host_config(self):
                 if self.client.option_minimal_versions[value]['supported']:
                     params[key] = getattr(self, value)
 
-        if self.restart_policy:
-            params['restart_policy'] = dict(Name=self.restart_policy,
-                                            MaximumRetryCount=self.restart_retries)
+        if self.client.docker_api_version < LooseVersion('1.22'):
```

Comment:
Please remove this. It doesn't hurt to set `restart_policy` to the correct value right away.

Reply:
Addressed with 3cdd6aba4cfa9296e2c0c3d752140edeb010fc54.